data/campaigns/Winds_of_Fate/scenarios/12_Epilogue.cfg

```diff
@@ -9,23 +9,14 @@
 
     [story]
         [part]
-            story= _ "Dominant Karron heard the Memories Spoken to her by the gliders. She lead several flights west to the Green Isle of the orcs. Striving together with the noble creatures of this far land, the infestation of orcs was destroyed so new eyries could be set there."
+            story= _ "Banished from the Greatland, Dominant Karron led several flights west. The next few flights that arrived on the Greatland reported that some drakes had joined her, planning to build new eyries overlooking the limitless prey of the Green Isle."
```

Comment:
Needs editing to keep Drake style but I see your point. I'll propose some change later, but is this one line important enough to mess with string freeze? At worst we can just put a change in 1.18.1? Which would be better?

Reply:
I think it's important enough to change during the freeze, and that the translators will understand a change that's done because it's feeling uncomfortably close to real-world racism for some of us.

The string freeze is permanent for 1.18 - the ideal aim is no further changes for the entire branch, but a more realistic aim is for a few fixes to be done before 1.18.0. Otherwise 1.18.0 comes out fully translated, we hopefully get new players, and we don't want a game that was already fully translated to suddenly show English again when the bug-fix releases come out.

Reply:
> a change that's done because it's feeling uncomfortably close to real-world racism for some of us.

Then why not just propose a small wording change which you are more comfortable with?

What frustrates me is that (during the string freeze) you are making a bigger and more subjective change to Karron's final character and the conclusions of the story. The Green Isle is never mentioned again after TRoW and the date of its liberation is left unspecified. So there is no harm in letting Karron have her conclusive win there after she has learned from mistakes and considers the advice of the gliders,

Reply:
When it's on one line, the amount of work for translators to handle a small change is roughly the same as for a large change. Unless it's one of the huge exposition paragraphs in UtBS, they're still likely to have to read the whole line, understand it, and then it doesn't matter that much how much they need to delete and retype.

Whether to leave stuff open-ended is already argued elsewhere in this PR.